explainer.md

```diff
@@ -156,9 +156,9 @@ for await (const stream of transport.incomingBidirectionalStreams) {
 const mediaSource = new MediaSource();
 await new Promise(resolve => mediaSource.addEventListener('sourceopen', resolve, {once: true}));
 const sourceBuffer = mediaSource.addSourceBuffer('video/webm; codecs="opus, vp09.00.10.08"');
-const transport = new WebTransport('/video');
+const wt = new WebTransport('/video');
```

Comment:
I'm not sure I understand this example, but maybe the constructor needs `{allowPooling: true}`?

Reply:
Done.